tests/Resources/Page/ODH/AiApps/Anaconda.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Documentation   Keyword suites to interact with Anaconda Commercial Edition
+Resource        ../../../Page/LoginPage.robot
+Library         SeleniumLibrary
+Library         JupyterLibrary
+
+
+
+*** Variables ***
+${ANACONDA_APPNAME}=  anaconda-ce
+${ANACONDA_DISPLAYED_NAME}=  Anaconda Commercial Edition
+${ANACONDA_KEY_IN}=  Anaconda CE Key
+${INVALID_KEY}=  abcdef-invalidkey
+${ERROR_MSG}=  error\nValidation failed\nError attempting to validate. Please check your entries.
+${VAL_SUCCESS_MSG}=  Validation result: 200
+${TOKEN_VAL_SUCCESS_MSG}=  Success! Your token was validated and Conda has been configured.
+
+
+*** Keywords ***
+Enable Anaconda
```

Comment:
This file is indented with 2 spaces instead of 4

Reply:
I haven't run robotidy. I'll do at the end after all the reviews and tests

Reply:
It's better that you lint the code before sending the PR, not just to ease our review but because if we approve it and then you send an final commit linting the code we'll have to approve it again before merging.

Reply:
done for Anaconda.resource and Rhosak.resource. For the dashboard TC file I see there is another PR applying robotidy changes so i didn't run it to overwrite the code to avoid conflicts.